components/edit/configure/imports.tsx

```diff
@@ -210,99 +202,114 @@ function isRemote(ref: string): boolean {
   );
 }
 
-async function getDisplayName(ref: string): Promise<string> {
-  let displayName: string =
-    ref.split('/').pop()?.replace('sys.', '').replace('.', ' ') ?? ref;
-
-  if (!ref.startsWith('sys.')) {
-    const loadedTool = await load(ref);
-    const loadedName = loadedTool.toolSet[loadedTool.entryToolId].name;
-    if (loadedName) {
-      displayName = loadedName;
-    }
-  }
-
-  return displayName.replace(/-/g, ' ');
-}
+const toolIconMap = new Map<string, () => React.ReactNode>([
+  [
+    'github.com/gptscript-ai/gpt4-v-vision@gateway',
+    () => <BsEye className="text-md" />,
+  ],
+  [
+    'github.com/gptscript-ai/dalle-image-generation@gateway',
+    () => <FaPaintBrush className="text-md" />,
+  ],
+  [
+    'github.com/gptscript-ai/answers-from-the-internet',
+    () => <GoGlobe className="text-md" />,
+  ],
+  [
+    'github.com/gptscript-ai/search-website',
+    () => <GoSearch className="text-md" />,
+  ],
+  ['github.com/gptscript-ai/browser', () => <GoBrowser className="text-md" />],
+  [
+    'github.com/gptscript-ai/tools/apis/slack/write',
+    () => <AiOutlineSlack className="text-md" />,
+  ],
+  [
+    'github.com/gptscript-ai/tools/apis/notion/write',
+    () => <SiNotion className="text-md" />,
+  ],
+  [
+    'github.com/gptscript-ai/tools/apis/trello',
+    () => <FaTrello className="text-md" />,
+  ],
+  [
+    'github.com/gptscript-ai/tools/apis/hubspot/crm',
+    () => <FaHubspot className="text-md" />,
+  ],
+  [
+    'github.com/gptscript-ai/tools/apis/outlook/mail/manage',
+    () => <PiMicrosoftOutlookLogoDuotone className="text-md" />,
+  ],
+  [
+    'github.com/gptscript-ai/tools/apis/outlook/calendar/manage',
+    () => <PiMicrosoftOutlookLogoDuotone className="text-md" />,
+  ],
+  [
+    'github.com/gptscript-ai/structured-data-querier',
+    () => <PiMicrosoftExcelLogo className="text-md" />,
+  ],
+  ['github.com/gptscript-ai/json-query', () => <SiJson className="text-md" />],
+  [
+    'github.com/gptscript-ai/context/filesystem',
+    () => <BsFiles className="text-md" />,
+  ],
+  [
+    'github.com/gptscript-ai/context/workspace',
+    () => <GoFileDirectory className="text-md" />,
+  ],
+  [
+    'github.com/gptscript-ai/tools/clis/github',
```

Comment:
@ryanhopperlowe this also needs to change to `'github.com/gptscript-ai/tools/apis/github`